app/views/guides/index.html.erb

```diff
@@ -20,13 +20,19 @@
   </div>
 
   <div class="col-md-9">
+    <p>
+      <%= link_to "Create a Guide" , new_guide_path, class: 'btn btn-primary' %>
+    </p>
+    <%= paginate @guides %>
     <table class='table table-bordered'>
       <thead>
         <tr class='table-header'>
-          <th>Title</th>
-          <th>Last updated by</th>
+          <th>Guide</th>
+          <th>Community</th>
```

Comment:
Now we have "Published by", "Content owner" and "Community". Should we stick with "Published by"?